web/components/contract/contract-tabs.tsx

```diff
@@ -94,6 +103,92 @@ export function ContractTabs(props: {
     (totalPositions > 0 ? `${shortFormatNumber(totalPositions)} ` : '') +
     maybePluralize('Holder', totalPositions)
 
+  const tabs = buildArray(
+    {
+      title: commentsTitle,
+      content: (
+        <CommentsTabContent
+          contract={mainContract}
+          comments={comments}
+          pinnedComments={pinnedComments}
+          setCommentsLength={setTotalComments}
+          blockedUserIds={blockedUserIds}
+          replyTo={replyTo}
+          clearReply={() => setReplyTo?.(undefined)}
+          className="-ml-2 -mr-1"
+          bets={uniqBy(bets.concat(betReplies), (b) => b.id)}
+          appRouter={appRouter}
+        />
+      ),
+    },
+    totalBets > 0 &&
+      (liveContract.mechanism === 'cpmm-1' ||
+        liveContract.mechanism === 'cpmm-multi-1') && {
+        title: positionsTitle,
+        content: (
+          <UserPositionsTable
+            key={liveContract.id}
+            positions={
+              // If contract is resolved, will have to refetch positions by profit
+              Object.values(userPositionsByOutcome).length > 0 &&
+              !liveContract.isResolved
+                ? userPositionsByOutcome
+                : undefined
+            }
+            contract={liveContract as BinaryContract}
+            setTotalPositions={setTotalPositions}
+          />
+        ),
+      },
+    totalBets > 0 && {
+      title: tradesTitle,
+      content: (
+        <Col className={'gap-4'}>
+          <BetsTabContent
+            key={liveContract.id}
+            contract={liveContract}
+            bets={bets}
+            totalBets={totalBets}
+            setReplyToBet={setReplyTo}
+          />
+        </Col>
+      ),
+    },
+    totalBets > 0 &&
+      liveContract.mechanism === 'cpmm-1' && {
+        title: 'Order book',
+        className: 'hidden md:block',
+        content: null,
+      }
+  )
+
+  const filteredTabs = tabs.filter(Boolean)
+
+  const orderBookTabIndex = filteredTabs.findIndex(
+    (tab) => tab.title === 'Order book'
+  )
+
+  const isOrderBookTabActive =
+    orderBookTabIndex >= 0 && activeIndex === orderBookTabIndex
+
+  const unfilledBets =
+    useUnfilledBets(liveContract.id, { enabled: isOrderBookTabActive }) ?? []
+
+  if (orderBookTabIndex >= 0) {
```

Comment:
what is this? Why isn't the order book simply in the tabs array as the `content` object?

Reply:
hmm I'm confused I thought this as well as the clause u asked about above in a comment made up the logic that makes it so useUnfilledBets only gets the unfilledbets once the conditions are met and prevent them from loading with the page